src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/SelectExpandNode.cs

```diff
@@ -157,6 +159,11 @@ public SelectExpandNode(SelectExpandClause selectExpandClause, IEdmStructuredTyp
         /// </summary>
         public IDictionary<IEdmNavigationProperty, SelectExpandClause> ExpandedNavigationProperties { get; private set; }
 
+        /// <summary>
+        /// Gets the list of EDM navigation properties to be expanded in the response.
+        /// </summary>
+        public IDictionary<IEdmNavigationProperty, ExpandedNavigationSelectItem> ExpandedNavigationPropertiesWithExpandedItem { get; private set; }
```

Comment:
>ExpandedNavigationPropertiesWithExpandedItem [](start = 81, length = 44)

what's the difference between ExpandedNavigationProperties and ExpandedNavigationPropertiesWithExpandedItem?

Reply:
ExpandedNavigationProperties has select expand clause as the value in the dictionary where as this has the entire ExpandedNavigationSelectItem in the dictionary. We require the ExpandedNavigationSelectItem for nested query options to appropriately generate the next link. 

---
In reply to: [282612794](https://github.com/OData/WebApi/pull/1724#discussion_r282612794) [](ancestors = 282612794)